src/components/ScholarshipsMadeSimple.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Container } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import Tab from '@material-ui/core/Tab';
+import TabPanel from '@material-ui/lab/TabPanel';
+import TabContext from '@material-ui/lab/TabContext';
+import TabList from '@material-ui/lab/TabList';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import PropTypes from 'prop-types';
+import LoginButton from './LoginButton';
+import HomeSection from './HomeSection';
+import pic from '../img/blank.png';
+
+const useStyles = makeStyles(() => ({
+  container: {
+    background: 'theme.palette.background.paper',
+    textAlign: 'left',
+  },
+  img: {
+    width: '100%',
+    height: '100%',
+    objectFit: 'contain',
+  },
+}));
+
+function ButtonList(user) {
+  return (
+    <>
+      {user === 'students' && <LoginButton />}
+      <Button
+        variant="outlined"
+        color="primary"
+        component={Link}
+        style={{ marginLeft: '20px' }}
+        to={user === 'students' ? '/scholarships' : '/scholarships/new'}
+        buttonProperties>
+        {user === 'students' ? 'Find Scholarships' : 'Submit A Scholarship'}
+      </Button>
+    </>
+  );
+}
+
+const content = {
+  0: {
+    direction: 'row',
+    tab: 'students',
+    title: 'Stress-free scholarships',
+    description:
+      'Et has minim eltir intellegat. Mea aeterno elefiend antiopam ad, nam no suscipit quaerendum. At name mininum ponderum. Est audiam animal molestiate te.',
+    button: ButtonList('students'),
+    picture: pic,
+  },
+  1: {
```

Comment:
0 and 1 aren't meaningful keys. I suggest either turning this dictionary into an array: 
```jsx
const tabs = [
   {...},
   {...},
];
```

Or attach more meaningful keys:
```jsx
const tabs = [
   'students': {...},
   'contributor': {...},
];
```